Homework 3/Homework 3/Program.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System.Text;
+
+namespace Homework_3
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            bool TrueFalse = false;
```

Comment:
It's better not to use such a way to name variables because it's not clear what this variable means.

Reply:
And the variable shouldn't start with a capital letter unless it is a property.